apm-sniffer/apm-sdk-plugin/cassandra-3.x-plugin/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-sdk-plugin</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>6.4.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-cassandra-3.x-plugin</artifactId>
+    <name>apm-cassandra-3.x-plugin</name>
+
+    <properties>
+        <compiler.version>1.8</compiler.version>
```

Comment:
If your plugin requires 1.8+ compiler, this plugin must be in optional-plugin. If your codes don't require 1.8, although the target driver is 1.8+, you don't need to set this.

Reply:
ok